src/routes/edit/share-options/exports/export-modal.tsx

```diff
@@ -143,8 +174,10 @@ export const ExportModal = () => {
 	}
 
 	const jsonCode: any = getFragmentJsonExportString(fragmentExportModal.fragment, fragments, styleClasses);
-	const reactCode: any = createReactApp(fragmentExportModal.fragment, fragments);
-	const angularCode: any = createAngularApp(fragmentExportModal.fragment, fragments);
+
+	const onVersionChange = (event: string) => {
```

Comment:
> const onVersionChange = (event: string) => {

Yes? No? It's generally not a good practice to pass inline functions as props due to reconciliation. When you pass a new function reference as a prop it can trigger unnecessary re-renders. 

If you're referring to deleting `reactcode`/`angularCode`, then yes it is necessary. We need to store them in state so when the values are updated it can trigger a re-render. I've tried without useState and the values of `reactCode`/`angularCode` remain `undefined` or whatever initial value you've set,